tests/sender/test_consumer.py

```diff
@@ -127,11 +127,11 @@ async def test_with_ssl(mocker):
     connect_robust.assert_called_once_with(
         URL(
             "amqp://rmq.example.com/%2Fvhost?"
-            "connection_name=FMN+sender+on+testdest"
-            "&auth=EXTERNAL"
+            "auth=EXTERNAL"
             "&cafile=/path/to/cacert"
             "&certfile=/path/to/certfile"
             "&keyfile=/path/to/keyfile"
             f"&no_verify_ssl={ssl.CERT_REQUIRED}"
+            "&connection_name=FMN+sender+on+testdest"
```

Comment:
Not sure about this 100%,but it looks as if we should instantiate the `URL` class of `aio_pika` (well, `yarl`) with a `query={"some": "dict"}` parameter instead of putting it all in one long string (which hopefully shouldn’t care about order of query params in the URL then).